pg_metadata/test/test_database_structure.py

```diff
@@ -79,7 +79,7 @@ def test_install_database(self):
         }
         alg = "{}:create_database_structure".format(provider.id())
         results = processing.run(alg, params, feedback=self.feedback)
-        self.assertEqual('0.0.3', results['DATABASE_VERSION'])
+        self.assertEqual('0.0.4', results['DATABASE_VERSION'])
```

Comment:
Thanks for reminding this one. It has been fixed on master. You can rollback this change and rebase on master.